apps/web/frontend/components/weekTable/weekTableTaskRow.tsx

```diff
@@ -13,17 +13,41 @@ export const WeekTableTaskRow = ({ interval, task }: WeekTableTaskRowProps) => {
     .map((workHour) => workHour.duration)
     .reduce((previous, current) => previous + current, 0)
 
+  const [{ data, error, fetching: projectsLoading }] = useMyProjectsQuery({
+    variables: { from: format(new Date(), 'yyyy-MM-dd'), filter: ProjectFilter.All },
+  })
```

Comment:
All the information we need (start and end date) exists in the parent component (`WeekTableProjectRowGroup`). Why not just pass it off as props?